docs/conf/portland/2022/cfp_email_templates.rst

```diff
@@ -0,0 +1,204 @@
+
+   In a few weeks I'll be in touch with some calendar invites for a sound check during the conference, to make sure you're all sorted with Hopin logins, audio and video, and to answer any questions you might have.
+
+   And I think that is it! I'm excited to see this taking shape and excited to see all of your hard work on the virtual stage!
+
+   Please get in touch if I can help with anything, if you have worries, thoughts or ideas.
+
+   Sam
+
+04 - Hopin URL and calendly invites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tickets:
+   {{cfp_variables['ticket']}}
+Calendly:
+   {{cfp_variables['calendly']}}
+
+----
+
+Subject:
+   WTD {{city}} {{year}} -- hopin and calendly invites
+
+::
+
+   Hi Folks,
+
+   The conference is almost upon us! First off, thank you all so much for all of the hard work you've put into getting the recordings done and uploaded.
+
+   Especially now the recordings are in, all of the hard work is behind you, and you can settle in and enjoy the conference. :-)
+
+   To that effect, please register your free Hopin speaker ticket so we we can connect the account to your talk:
+
+   ➡️ [register your free Hopin speaker ticket]({{cfp_variables['ticket']}})
+
+   The entire conference is held in Hopin, from writing day on Sunday, all talks, unconference sessions, chat, etc. Only the Monday evening social will be held in Spatial.chat.
+
+   I'd also like you to schedule an audio-visual check with me at least **an hour before** your talk. This helps me know you're around 😉 and lets us clear up any audio or visual issues before your Q&A. We recommend you do this even if you're confident about your setup. The audio-visual checks happen in a private Hopin room.
+
+   ➡️ [book an audio-visual check]({{cfp_variables['calendly']}})
+
+   Remember, all Q&A sessions **start at the end of your recording**, in the Speaker Q&A session (the Monday and Tuesday Q&A sessions have different URLs).
+
+   Some speakers like to hang out in the event chat and answer questions during their talk, but that is entirely up to you. We'll be moderating questions regardless, and then the Q&A will be a live video call with the MC.
+
+   If you have any questions at all about the event, you definitely know how to reach me by now.
+
+   Sam
+
+05 - Post conf
+~~~~~~~~~~~~~~
+
+Feedback:
+   {{cfp_variables['feedback_form']}}
+Gift:
+   {{cfp_variables['speaker_gift']}}
+
+----
+
+Subject:
+   WTD {{city}} {{year}} -- feedback, hoodies and THANKS
+
+::
+
+   Hey Folks,
+
+   It is a wrap! Thank you one last time for your hard work, insight and creativity.
+
+   We'll be publishing the videos at some point this week or next, keep an eye on Twitter, Slack or the mailing list for those. 
+
+   We'd love to know how you found the whole process, from A to Z so we can improve next time. To that end we've got an anonymous (keep in mind that there aren't so many speaks) feedback form for you here:
+
+   ➡️ [Speaker feedback form]({{cfp_variables['feedback_form']}})
```

Comment:
Feels like there are multiple styles in the emails for task lists. This emote, a normal list, a list with ascii checkboxes. Don't know if that's on purpose.

Reply:
Could probably use some tweaking yeah.